src/main/java/thePackmaster/potions/PackInAJar.java

```diff
@@ -0,0 +1,82 @@
+package thePackmaster.potions;
+
+
+import basemod.abstracts.CustomPotion;
+import com.badlogic.gdx.graphics.Color;
+import com.megacrit.cardcrawl.actions.common.DiscardAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.PowerTip;
+import com.megacrit.cardcrawl.localization.PotionStrings;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.actions.EasyModalChoiceAction;
+import thePackmaster.cards.PackmasterModalChoiceCard;
+import thePackmaster.packs.AbstractCardPack;
+import thePackmaster.util.Wiz;
+
+import java.util.ArrayList;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+
+public class PackInAJar extends CustomPotion {
+    public static final String POTION_ID = SpireAnniversary5Mod.makeID("PackInAJar");
+    private static final PotionStrings potionStrings = CardCrawlGame.languagePack.getPotionString(POTION_ID);
+    public static final String NAME = potionStrings.NAME;
+    public static final String[] DESCRIPTIONS = potionStrings.DESCRIPTIONS;
+
+    public PackInAJar() {
+        super(NAME, POTION_ID, PotionRarity.UNCOMMON, PotionSize.GHOST, PotionColor.NONE);
```

Comment:
This could be rare, and in my opinion it's powerful enough that it should be. Getting a whole hand of cards is a really unique effect. I'm happy to have this potion exist as a very powerful one-time-use resource that will often make a decisive difference even in the hardest fights -- but if I'm assessing the impact this will have correctly, it fits better with other rare potions.

And assuming we make Booster Brew a Packmaster-specific potion, I think that fits better as Packmaster's uncommon potion, leaving Pack Rip and Smithing Oil to fight it out for the rare slot. Not saying we need to cut one, though as I mentioned in another comment I think we should discuss whether we want to keep Smithing Oil (I'm fine if we do).